Route/Customer/FproCustomer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: remi
+ * Date: 1/20/17
+ * Time: 6:10 PM
+ */
+
+namespace monsieurgourmand\Bundle\InterfaceBundle\Route\Customer;
+
+use monsieurgourmand\Bundle\InterfaceBundle\Model\Customer;
+use monsieurgourmand\Bundle\InterfaceBundle\Service\MGD;
+
+class FproCustomer
+{
+    private $entity;
+    private $url;
+    private $master;
+
+    public function __construct(Customer $customer)
+    {
+        $this->master = $customer->getMaster();
+        $this->entity = \monsieurgourmand\Bundle\InterfaceBundle\Model\FproCustomer::class;
+        $this->url = "/customers/" . $customer->getIdCustomer() . "/fprocustomers";
+    }
+
+    public function getAll($params = [], $format = MGD::FORMAT_OBJECT)
```

Comment:
Je mettrais bien doc php un @return FproCustomer[]
Voire un
```suggestion
    public function getAll($params = [], $format = MGD::FORMAT_OBJECT): array
```